kitsune/wiki/views.py

```diff
@@ -200,6 +200,12 @@ def document(request, document_slug, template=None, document=None):
     # The list above was built backwards, so flip this.
     breadcrumbs.reverse()
 
+    browser = get_browser(request)
+    if product.slug == 'thunderbird' and browser != 'Firefox':
+        fx_download_button = True
+    else:
+        fx_download_button = False
```

Comment:
Things of the form

```
if foo == bar:
    baz = True
else:
    baz = False
```

are better written as `baz = foo == bar`. Sometimes parenthesis help readability, like `baz = (foo == bar)`, but they aren't required.